packages/ui/nodes/image/src/ImageElement/ImageImg.tsx

```diff
@@ -0,0 +1,25 @@
+import { TImageElement } from '@udecode/plate-image';
+import { createComponentAs } from '../utils/createComponentAs';
+import { createElementAs } from '../utils/createElementAs';
+import { HTMLPropsAs } from '../utils/types';
+import { useElement } from '../utils/useElement';
+import { useImageCaptionState } from './ImageCaption';
+
+export const useImageImg = (props?: HTMLPropsAs<'img'>): HTMLPropsAs<'img'> => {
+  const { url } = useElement<TImageElement>();
+
+  const { captionString } = useImageCaptionState();
+
+  return {
+    src: url,
+    alt: captionString,
+    draggable: true,
```

Comment:
Does this make the entire image draggable and not just the handle?

Reply:
Yes, overridable ofc.